starter-code/public/admin.html

```diff
@@ -49,11 +54,13 @@
     <script src="/vendor/scripts/handlebars.js"></script>
     <script src="/scripts/article.js"></script>
     <script src="/scripts/articleView.js"></script>
-    <!-- TODO: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <!-- DONE: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <!-- Estimate 10 min |  Actual 7 min-->
     <script>
-      // TODO: After we fetch and load all the articles, what view function do we want to run next?
+      // ????: After we fetch and load all the articles, what view function do we want to run next?
       // Let's pass in a callback function, to determine what happens AFTER all articles are loaded.
       Article.fetchAll();
```

Comment:
I am getting an error here in the console: Uncaught ReferenceError: Article is not defined
    at admin.html:62